internal/otelcollector/config/otlpexporter/config_builder.go

```diff
@@ -74,6 +75,32 @@ func ExporterID(output *telemetryv1alpha1.OtlpOutput, pipelineName string) strin
 	return fmt.Sprintf("%s/%s", outputType, pipelineName)
 }
 
+func makeTLSConfig(output *telemetryv1alpha1.OtlpOutput, otlpEndpointValue, pipelineName string) config.TLS {
+	var cfg config.TLS
+	cfg.Insecure = isHTTPEndpoint(otlpEndpointValue)
+
+	if output.TLS == nil {
+		return cfg
+	}
+
+	// endpoint scheme of https or http has precedence over configured insecure option
+	if !(isHTTPSEndpoint(otlpEndpointValue) || isHTTPEndpoint(otlpEndpointValue)) {
```

Comment:
Is there a good reason to check additionally for the https prefix to enable TLS? I remember that it is the default for GRPC when not having any prefix. We should not change this behaviour without a good reason.